interactions/console.py

```diff
@@ -3,70 +3,55 @@
 class Console:
 
     @staticmethod
-    def is_int(stri):  # czy można zmienić w int
+    def is_int(string_to_check):  # Sprawdza czy dany string można zmienić w int, True na tak False na nie
         try:
-            a = int(stri)
+            a = int(string_to_check)
             return True
         except ValueError:
             return False
 
     @staticmethod
-    def clear_view():  # czyszczenie ekranu TYLKO NA LINUXS!!!
+    def clear_view():  # Czyszczenie ekranu DZIAŁA TYLKO NA LINUXS!!!
         os.system('clear')
 
     @staticmethod
-    def output(text):  # Print w funkcji
+    def output(text):  # Pisze wysłany tekst
         print(text)
 
     @staticmethod
-    def input_text(entry):  # Wzięcie string i return
-        text = input(str(entry))
-        return text
+    def input_text(question):  # Zadaje pytanie i zwraca odpowiedź
+        entry = input(str(question))
+        return entry
 
     @staticmethod
-    def input_number(stri, rang):  # Czy jest to liczba w  liście
-        entry = input(stri)
+    def input_number(question, posible_numbers):  # Zadaje pytanie i zwraca liczbę z przesłanej listy
+        entry = input(question)
         while True:
             if Console.is_int(entry):
-                numer = int(entry)
-                is_in_range = False
-                for x in rang:
-                    if x == numer:
-                        is_in_range = True
-                if is_in_range:
-                    return numer
+                number = int(entry)
+                if number in posible_numbers:
+                    return number
                 else:
-                    a = ['Ten numer nie nalerzy do listy. Podaj numer należący do listy: ', str(rang)]
-                    a = ''.join(a)
-                    entry = input(a)
-
+                    entry = input(f'Ten numer nie nalerzy do listy. Podaj numer należący do listy: {str(posible_numbers)}')
             else:
-                a = ['To nie jest numer. Podaj numer należący do listy: ', str(rang)]
-                a = ''.join(a)
-                entry = input(a)
+                entry = input(f'To nie jest numer. Podaj numer należący do listy: {str(posible_numbers)}')
 
     @staticmethod
-    def input_numer_in_range(stri, beginr, endr):  # czy jest to liczba pomiędzy tymi
-        entry = input(stri)
+    def input_numer_in_range(question, beginr, endr):  # Zadaje pytanie i oddaje liczbę w danym zakresie

```

Comment:
+ beginr i endr to nie są dobre nazwy, i 'number' a nie 'numer'

+ i nie wiem czy takie rozwiązanie będzie dobre, ale jeśli spojrzeć na funkcje 'input_number' i 'input_number_in_range' to okaże się że są niemal identyczne więc funkcję 'input_number_in_range' można przerobić jakoś tak:

![obraz](https://github.com/wikimody/Quarto-project/assets/150141561/81ed4dd6-a8b4-4747-a4e4-c2b9ecb6ab31)

ale można zostawić tak jak jest

Reply:
Akurat beginr i endr poprawiłem ale zapomniałem to skomitować

Reply:
I zwyczajnie nie ma sensu zmieniać number_in_range bo jest gotowa i działająca funkcja tam więc nic nie oszczędzamy na tej zmianie